packages/zoltan2/test/partition/taskMappingTest3.cpp

```diff
@@ -400,7 +400,10 @@ int main(int argc, char *argv[]){
         //hopper[2] = 24;
         part_t *machineDimensions = NULL;
         //machineDimensions = hopper;
+        Teuchos::RCP <Zoltan2::Environment> env
+          = rcp(new Zoltan2::Environment(comm));
```

Comment:
In general, we don't want to force users to create the environment.  I propose returning to original coordinateTaskMapperInterface, and using the problemComm as the communicator to create the needed environment.